client/App.vue

```diff
@@ -40,6 +63,17 @@ export default {
         }
       }
     },
+    onEnvironmentSelectChange(environment) {
+      if (environment !== this.environment.value) {
```

Comment:
To make the code easier to read, consider using early returns.

Reply:
no problem